requirements.txt

```diff
@@ -1,3 +1,4 @@
 Click==7.1.2
 tweepy==3.9.0
-ujson==3.1.0
\ No newline at end of file
+ujson==3.1.0
+pytz==2020.1
```

Comment:
Is this requirement still needed? I don't see it used in any of the affected files